### PR TITLE
Fix Few issues related to ballerina commands

### PR DESCRIPTION
**Pull request by warunalakshitha**
## Purpose
Fix https://github.com/ballerina-platform/ballerina-distribution/issues/252
Fix https://github.com/ballerina-platform/ballerina-distribution/issues/248
Fix https://github.com/ballerina-platform/ballerina-distribution/issues/253
Fix https://github.com/ballerina-platform/ballerina-distribution/issues/267
Fix https://github.com/ballerina-platform/ballerina-distribution/issues/269

### CODE REVIEW

File: ballerina-command/src/main/java/org/ballerinalang/command/util/ToolUtil.java [hunk 1]
```diff
@@ -565,9 +558,9 @@ public static void handleInstallDirPermission() {
         try {
             String installationPath = OSUtils.getInstallationPath();
             if (!new File(installationPath).canWrite()) {
-                throw ErrorUtil.createCommandException("current user does not have write permissions.\n\n" +
+                throw ErrorUtil.createCommandException("current user does not have permissions to run the command\n\n" +
                                                                "Current user does not have write permissions to " +
-                                                               "ballerina installation path : " + installationPath +
+                                                               "ballerina installation path " + installationPath +
```
Comment (keizer619) [inline ballerina-command/src/main/java/org/ballerinalang/command/util/ToolUtil.java]: Ballerina reference has to be "Ballerina"

File: ballerina-command/src/main/java/org/ballerinalang/command/cmd/PullCommand.java [hunk 2]
```diff
@@ -49,15 +49,30 @@ public void execute() {
...
-        } else if (pullCommands.size() > 1) {
+        }
+
+        if (pullCommands.size() > 1) {
             throw ErrorUtil.createUsageExceptionWithHelp("too many arguments given");
         }
+        ToolUtil.handleInstallDirPermission();
+        PrintStream printStream = getPrintStream();
+        String distribution = pullCommands.get(0);
+        String distributionType = distribution.split("-")[0];
```
Comment (Kishanthan) [inline ballerina-command/src/main/java/org/ballerinalang/command/cmd/PullCommand.java]: What if the dist name has completely wrong name? Have we handled that? Looks like this line will break in that case.

File: ballerina-command/src/main/java/org/ballerinalang/command/cmd/FetchCommand.java [hunk 3]
```diff
@@ -49,13 +49,32 @@ public void execute() {
             return;
         }
 
-        if (fetchCommands == null) {
+        if (fetchCommands == null || fetchCommands.size() == 0) {
```
Comment (Kishanthan) [inline ballerina-command/src/main/java/org/ballerinalang/command/cmd/FetchCommand.java]: We thought not to include fetch command for this release. Shall we remove it? 

We can keep this class, but remove where this is instantiated and mentioned in help text, etc.

File: ballerina-command/src/main/java/org/ballerinalang/command/cmd/PullCommand.java (hunk 2 above)
Comment (warunalakshitha) [reply ballerina-command/src/main/java/org/ballerinalang/command/cmd/PullCommand.java]: You can get "" still split will not fail. null values are handled in upper code.

File: ballerina-command/src/main/java/org/ballerinalang/command/cmd/FetchCommand.java (hunk 3 above)
Comment (keizer619) [reply ballerina-command/src/main/java/org/ballerinalang/command/cmd/FetchCommand.java]: Lets remove from the main method and keep these logics

Comment (warunalakshitha) [reply ballerina-command/src/main/java/org/ballerinalang/command/cmd/FetchCommand.java]: Will remove those in next PR.

File: ballerina-command/src/main/java/org/ballerinalang/command/util/ToolUtil.java (hunk 1 above)
Comment (warunalakshitha) [reply ballerina-command/src/main/java/org/ballerinalang/command/util/ToolUtil.java]: Already fixed in my next PR.

**Pull request merged**